pkg/render/kube-controllers.go

```diff
@@ -558,3 +609,10 @@ func kubeControllersVolumes(defaultMode int32, managerSecret *v1.Secret) []v1.Vo
 
 	return []v1.Volume{}
 }
+
+func randomString(length int) string {
```

Comment:
An identical func already exists in rand.go

Reply:
Or even better, one that doesnt need to be urlencoded in crypto.go.

I guess you can use that one and remove the other two functions as they are unused.